tests/framework/PostProcessors/BasicStatistics/print_xml.xml

```diff
@@ -86,7 +88,7 @@
   <Samplers>
     <MonteCarlo name="MC_external">
       <samplerInit>
-        <limit>4</limit>
+        <limit>1000</limit>
```

Comment:
Note this change had minimal impact on the run time, but vastly increases the reliability.